core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -56,6 +56,15 @@ trait BillingProfileManagerDAO {
 
   def getAllBillingProfiles(ctx: RawlsRequestContext)(implicit ec: ExecutionContext): Future[Seq[ProfileModel]]
 
+  @throws[ApiException]
```

Comment:
I don't think I've seen this annotation before. Should all of our DAO methods be including this?

Reply:
It was needed to mock out throwing an exception. It seems like there are other places where it's not needed to do the same thing, but I assume that the other DAO methods don't include it because there's no test where we mock out an exception for those methods. I'm not inclined to add it to the other methods preemptively, but I could be convinced